selene-lib/src/lints/roblox_incorrect_roact_usage.rs

```diff
@@ -17,6 +17,23 @@ use if_chain::if_chain;
 
 pub struct IncorrectRoactUsageLint;
 
+fn get_lua_table_key_format(expression: &ast::Expression) -> String {
+    match expression {
+        ast::Expression::Value { value, .. } => match &**value {
+            ast::Value::String(token) => {
+                let string = token.to_string();
+                if string.contains(" ") {

```

Comment:
Not just spaces. Needs to be a valid identifier.

Test cases:

"A-B-C", should become []
"ABC", should become ABC
"ABC123", should become ABC123
"123ABC", should become []

Reply:
Sorry, realized that after requesting review, fixed in https://github.com/Kampfkarren/selene/pull/508/commits/58fa9ee22d532d0f37e659b48da2c2bbb5562de7.